common-legacy/src/main/thrift/CentralDogma.thrift

```diff
@@ -178,6 +188,7 @@ struct Project {
 enum QueryType {
     IDENTITY = 1,
     JSON_PATH = 2,
+    JSON_MERGER = 3,
```

Comment:
I don't think we need this. `QueryType` of `mergeFiles` should be:

- `IDENTITY` if no JSON paths are given.
- `JSON_PATH` if one or more JSON paths are given.

Reply:
I found while I'm fixing this that the supported entry types for `IDENTITY` are `TEXT` and `JSON`. Currently, we only support `JSON` for merge. Is it OK to use `IDENTITY`?